frontend/src/components/dashboard/create/create-modal/CreateModal.tsx

```diff
@@ -99,114 +76,104 @@ const CreateModal = ({
 
           <div className={styles.channelTypes}>
             <div className={styles.channelContainer}>
-              <PrimaryButton
-                className={cx(styles.button, {
-                  [styles.active]: selectedChannel === ChannelType.SMS,
-                })}
-                onClick={
-                  selectedChannel === ChannelType.SMS
-                    ? undefined
-                    : () => setSelectedChannel(ChannelType.SMS)
-                }
-              >
-                SMS
-                <i
-                  className={cx(
-                    'bx',
-                    styles.icon,
-                    channelIcons[ChannelType.SMS]
-                  )}
-                ></i>
-              </PrimaryButton>
-
+              <input
+                type="radio"
+                aria-label={ChannelType.Email}
+                value={ChannelType.Email}
+                checked={selectedChannel === ChannelType.Email && !protect}
+                onChange={() => {
+                  setSelectedChannel(ChannelType.Email)
+                  setProtected(false)
+                }}
+              />
+              <p className={styles.subtext}>Email</p>
```

Comment:
Should use `<label />` tag and `htmlFor` prop so that user can click on the label to choose the option as well